### DISCUSSION TITLE
Amend Markdown help page with chapter "Collapsible code block (aka spoiler)"

### DISCUSSION THREAD

**Discussion by porg** [community/community#22448] [Projects and Issues]
<p>In 2016 there was a request “Collapsible contents (code block) in comments / spoiler tag” and a <a href="https://github.com/dear-github/dear-github/issues/166#issuecomment-236342209" rel="noopener nofollow ugc">comment</a> showed that this was possible already back then with the HTML syntax:</p>
<p><code>&lt;details&gt;&lt;summary&gt;Click to expand&lt;/summary&gt;Full content here.&lt;/details&gt;</code></p>
<p>Instead of this valuable info being buried deep in an issue conversation, please add it to the official <a href="https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax" rel="noopener nofollow ugc">Syntax Reference on Github-flavored Markdown</a>. This is linked to by the hint “Styling with Markdown is supported” below the UI for writing issues and comment, so the first stop for users informing themselves about the syntax.</p>
<p>And the ability to add longer code blocks without harassing your fellow readers with long scrolls is a legit syntax need, as the popularity of the issue shows.</p>
<p>So please to the chapter “Quoting code” add a sub-chapter “Collapsible code block (aka spoiler)” and there add this valuable information. Thanks.</p>

**Comment by porg** [comment]
<p>Could I please get a reaction? At least a first reaction. Thanks.</p>

**Comment by porg** [comment, marked as answer]
<p><strong>Closing:</strong> Ammending the documentation with this hint will not happen.<br>
GitHub representative <a class="mention" href="/mylesborins">@MylesBorins</a> answered me elsewhere <a href="https://github.com/github/feedback/discussions/5979#discussioncomment-1479071" rel="noopener nofollow ugc">why not</a>:</p>
<blockquote>
<p>The reason it isn’t documented is because it isn’t GitHub flavored markdown… it is simply inline HTML. There isn’t GitHub flavored markdown specifically for <code>details</code>. We don’t document HTML features.</p>
</blockquote>

**Comment by ghost** [comment]
well, it seems to be there now
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections

**Reply by porg** [reply]
Glad to see it's finally in official docs and not just hints in some forums or issue-discussions.
